enhancements/authentication/configuring-webhook-token-authenticators.md

```diff
@@ -0,0 +1,159 @@
+---
+title: configuring-webhook-token-authenticators
+authors:
+  - "@stlaz"
+reviewers:
+  - "@sttts"
+  - "@deads2k"
+approvers:
+  - "@deads2k"
+  - "@sttts"
+  - "@mfojtik"
+creation-date: 2019-12-10
+last-updated: 2019-12-10
+status: implementable
+see-also:
+  - ""
+replaces:
+  - ""
+superseded-by:
+  - ""
+---
+
+# Configuring Webhook Token Authenticators
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions [optional]
+
+  1. Should we leverage the `webhookTokenAuthenticators` field of the `authentications.config.openshift.io`
+     resource or should we use CRDs instead?
+  3. Will any changes be needed for the console?
```

Comment:
to use the webhook?  none expected, they're all external for the console.

Reply:
> o use the webhook? none expected, they're all external for the console.

move this to the appropriate section with explanation.

Reply:
I omitted this part as there indeed does not seem to be much to comment on